app/src/main/java/com/winlator/contents/ContentsManager.java

```diff
@@ -191,7 +195,7 @@ public void extraContentFile(Uri uri, OnInstallFinishedCallback callback) {
             }
         }
 
-        if (profile.type == ContentProfile.ContentType.CONTENT_TYPE_WINE) {
+        if (profile.type == ContentProfile.ContentType.CONTENT_TYPE_WINE || profile.type == ContentProfile.ContentType.CONTENT_TYPE_PROTON) {
```

Comment:
Will need to do regression here, but it's required for the custom proton builds as they identify as "proton" and not "wine"